CartoDB/cartodb
# Moving to node 6.9.2 and npm 3.10.9

Related ticket: https://github.com/CartoDB/cartodb/issues/11935

It will force us to have the same version of node (v6.9.2) and npm (v3.10.2) as we have in our staging and production servers.

And many thanks to shrinkwrap specialist @rochoa, who has make it possible ❤️ .

## How to test it
- Install node `6.9.2`, if you use nvm, it will install the necessary npm version, so safe and sound.
- First thing, check that `npm run update-internal-deps` works in your machine.
- Run your local CARTO environment and check if things work like the have to.
- Then check the same in staging, check deploy times and compare with a previous one.

cc @CartoDB/frontend because it will force us to use the same version of node and npm for frontend development. And also is great to share all the black magic @rochoa has done in order to keep using shrinkwrap :dancer:.
